Source/WebKit/Shared/RemoteLayerTree/DynamicContentScalingImageBufferBackend.mm

```diff
@@ -128,6 +128,10 @@ static CFDictionaryRef makeContextOptions(const DynamicContentScalingImageBuffer
                 String objectDescription { adoptCF(CFCopyDescription(port)).get() };
                 auto description = makeString("Unexpected type in DCS ports array: "_s, typeID, " "_s, typeDescription, " "_s, objectDescription);
                 logAndSetCrashLogMessage(description.utf8().data());
+
+                std::array<uint64_t, 6> values { 0, 0, 0, 0, 0, 0 };
+                strncpy(reinterpret_cast<char*>(values.data()), typeDescription.utf8().data(), sizeof(values));
```

Comment:
Maybe obvious but I did't spot it right away, how do we know typeDescription.utf8().data() is at least 48 bytes?

Reply:
we don't, but why do we care? it's a null-terminated string and we're using a string function; if it's shorter strncpy will bail

Reply:
Oh yes its not memcpy, please ignore